spec/registration_server_spec.rb

```diff
@@ -9,11 +9,13 @@
   let(:port) { 9000 }
   let(:http_server) { double("http_server") }
   let(:certs_factory) { double("certs factory") }
+  let(:updates_url) { "https://updates.suse.com/sles12" }
```

Comment:
The constructor is documented to want a URI but we are passing a String. Also, the PR is about switching to HTTP but we test with HTTPS. Is that OK?